test/thin6d_ions/fort.3

```diff
@@ -75,7 +75,8 @@ HION
 208 82 1.93687690162648124E+02 82
 NEXT
 DIST
-READ initial.dat
+! FORMAT ID none none x[m] y[m] none xp[rad] yp[rad] none ion_a ion_z mass[GeV] P[GeV] dT[s]
```

Comment:
If xp[rad] is Px/P the units are not radians.

Reply:
This is a comment, will delete the line.